pilot/pkg/networking/core/v1alpha3/listener_builder_test.go

```diff
@@ -571,6 +628,7 @@ spec:
 `
 
 func TestInboundListenerFilters(t *testing.T) {
+	test.SetForTest(t, &features.EnableDualStack, true)
```

Comment:
I do not see any changes in listener.go which is based on feature flag. Then why do you set the feature flag for this test? If this is really needed, you should be adding a separate test for dual-stack. Otherwise even the existing tests get run with the feature flag enabled, which is not desired.

Reply:
Please check the `Endpoint: &model.IstioEndpoint{` in `ServiceInstance`. I separated this test into 2 test cases: `TestInboundListenerFilters` and `TestInboundListenerFiltersWithMulAddrsIstioEndpoint`